webapp/src/locales/zh_CN.ts

```diff
@@ -0,0 +1,75 @@
+
```

Comment:
I would recommend handling Chinese support in a follow-up.

Reply:
Okay